change/@fluentui-react-819f64b3-508d-46da-8cec-303110a17ede.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "feat: Handle when FocusTrapZone wraps an iframe",
```

This is technically a fix now

```suggestion
  "comment": "fix: Handle when FocusTrapZone wraps an iframe",
```